packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,51 @@
+
+* A shorter and restrictive session is required for privileged access apps, such as an e-commerce site where users make payments.
+* A longer and less restrictive session is typical for non-privileged access apps, such as a catalog site where users browse listed items. The session for these apps can remain active even when the user navigates away, closes the browser tab or website, or closes the mobile app.
+
+Identity management admins and app developers need to collaborate on how to manage the IdP session with the application sessions to produce an optimal user experience with security, device, and app privileged access in mind. This implies balancing terminating sessions and redirecting the user to re-authenticate for security purposes, combined with extending the session or silently re-authenticating with a new session for a frictionless user experience.
+
+### Examples of sign out experiences
+
+#### Privileged access use case
+
+For privileged access apps, such as an e-commerce payment app, securing sensitive user information is paramount and application sessions need to be tightly restricted. Application sessions are usually short lived and inactivity timeouts are in the minutes for these types of apps. Developers need to employ best practices and end sessions when they are no longer required. If the application session is terminated, the user must be redirected to the IdP to re-authenticate back into the app for a new session. If there is no active IdP session during the redirect, the user is prompted to sign in to the IdP and then use SSO to sign in to the app for a new session.
+
+#### Non-privileged access use case
+
+For non-privileged access apps, such as a loyalty program catalog, sessions can be active for a long period of time (even days) and app developers don't need to implement session timeouts to restrict access. This is especially true for mobile apps where you want your users to have a seamless and frictionless user experience. In this scenario, the application session can outlive the IdP session. Application sessions can still track user interactions without the IdP session. However, when the user signs out and ends the application session, without the IdP session, the user is redirected to the IdP sign-in page and must authenticate with the IdP to gain access to the app, as before.
+
+### Terminate sessions
+
+#### Local logout
+
+Local logout is the act of signing the user out of the app and terminating the application session. The user IdP session is still active and other application sessions aren't affected by terminating a specific local application session. Local logout is managed by the app. If the user wants to access the app again, they are automatically authenticated through the active IdP session into the app and the app creates a new session for the user.
+
+See [Sign users out of your app](/docs/guides/sign-users-out/-/sign-out-of-your-app/) for an example of a Local logout implementation with Okta.
```

Comment:
would it be Local logout, Local Logout, or local logout here?

Reply:
Thanks, I'll update to "local logout".